services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/_core.py

```diff
@@ -476,8 +465,8 @@ async def _update_service_spec(
                         e.status == status.HTTP_500_INTERNAL_SERVER_ERROR
                         and "out of sequence" in e.message
                     ):
-                        raise TryAgain() from e
-                    raise e
+                        raise TryAgain from e
```

Comment:
again here using class instead of an instance.

Reply:
see [https://beta.ruff.rs/docs/rules/unnecessary-paren-on-raise-exception/](https://beta.ruff.rs/docs/rules/unnecessary-paren-on-raise-exception/) although I do not really care. I have to say I find it more concise, what is sure is that it is for sure not wrong.